webapps/api/backend/views/vdisks.py

```diff
@@ -171,9 +171,10 @@ def create(self, devicename, size, vpool_guid, storagerouter_guid):
         storagerouter = StorageRouter(storagerouter_guid)
         for storagedriver in storagerouter.storagedrivers:
             if storagedriver.vpool.guid == vpool_guid:
-                return VDiskController.create_new.delay(diskname=devicename,
-                                                        size=size,
-                                                        storagedriver_guid=storagedriver.guid)
+                return VDiskController.create_new.s(diskname=devicename,
```

Comment:
Declined commit as discussed on Skype chat
Task should not be routed. In the lower layer the vdiskcontroller should route the command itself